yarn/test_yarn.py

```diff
@@ -159,3 +218,18 @@ def test_check(self, mock_requests):
             self.assertMetric(metric,
                 value=value,
                 tags=self.YARN_NODE_METRICS_TAGS)
+
+        # Check the YARN Root Queue Metrics
+        for metric, value in self.YARN_ROOT_QUEUE_METRICS_VALUES.iteritems():
+            self.assertMetric(metric,
+                value=value,
+                tags=self.YARN_ROOT_QUEUE_METRICS_TAGS)
+
+        # Check the YARN Custom Queue Metrics
+        for metric, value in self.YARN_QUEUE_METRICS_VALUES.iteritems():
+            self.assertMetric(metric,
+                value=value,
+                tags=self.YARN_QUEUE_METRICS_TAGS)
+
+        with self.assertRaises(Exception):
+            self.assertMetric('yarn.queue.absolute_capacity', value=23, tags=self.YARN_QUEUE_NOFOLLOW_METRICS_TAGS)
```

Comment:
Could you add a comment explaining that we are testing the `filter_queues` here ?